src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -1299,8 +1299,8 @@ public void should_return_a_200_with_multiple_tasks_in_search_results_when_avail
 
         //Claim a task to set state to assigned
         given.iClaimATaskWithIdAndAuthorization(taskId3,
-                                                caseworkerCredentials.getHeaders(),
-                                                HttpStatus.NO_CONTENT);
+            caseworkerCredentials.getHeaders(),
+            HttpStatus.NO_CONTENT);
```

Comment:
Are these formatting changes required?

Reply:
no looks like I have a different checkstyle. will fix it